docs/content/v2.14/explore/change-data-capture/debezium-connector-yugabytedb.md

```diff
@@ -44,7 +44,7 @@ Debezium supports databases with UTF-8 character encoding only. With a single-by
 
 ## Release series 1.9.5.y
```

Comment:
Why is this section not in preview and stable ? Can you add it if its supported in 2.19 and 2.18?

Reply:
Section completely revamped for 2.18. Address in another ticket.